pkg/reconciler/dependencybuild/dependencybuild.go

```diff
@@ -507,6 +503,11 @@ func (r *ReconcileDependencyBuild) handleStateContaminated(ctx context.Context,
 func createLookupBuildInfoTask(build *v1alpha1.DependencyBuildSpec) *pipelinev1beta1.TaskSpec {
 	image := os.Getenv("JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE")
 	recipes := os.Getenv("RECIPE_DATABASE")
+	path := build.ScmInfo.Path
+	//TODO should the buidl request process require context to be set ?
```

Comment:
possibly better ways to address an empty context (in the java code or elsewhere) give me pause from merging this even with failing in-repo e2e's in order to facilitate debug /resolution by the rest of the team wrt not getting at least some artifactbuilds or dependencybuilds to complete state.

Reply:
but if need be let's go with this and the correct it in a follow up PR that ideally addresses the remaining issues preventing at least some of the artifactbuilds and dependencybuilds from reaching complete state